### PR TITLE
Pract.07:  Add Task01-Task03 | Viktoria Koleva

### PR DESCRIPTION
**Pull request by Viktoria035**
Viktoria Koleva
6 5MI0600139

### CODE REVIEW

File: SI/Sem.07/Pract.07/References/Task03.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		std::cout << mid << " " << max << " " << min;
+	}
+	else if (max < min && min < mid)
+	{
+		std::cout << max << " " << min << " " << mid;
+	}
+	else if (max < mid && mid < min)
+	{
+		std::cout << max << " " << mid << " " << min;
+	}
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Pract.07/References/Task03.cpp:30]: Can be optimized
if a < b => swap
then if b < c => swap
then if a < b => swap

File: SI/Sem.07/Pract.07/References/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	}
+}
+int main()
+{
+	char symbol;
+	std::cin >> symbol;
+	if (symbol >= LOWER_BORDER_TO_SMALL && symbol <=LOWER_BORDER)
+	{
+		toUpper(symbol);
+	}
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Pract.07/References/Task02.cpp:35]: You are doing the same check in the toUpperFunction
The same for the code below with the toLower
Upload only the two function - toUpper and toLower!

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

File: SI/Sem.07/Pract.07/References/Task01.cpp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+void swap(int* a, int* b)
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/Task01.cpp:2]: why not references?

File: SI/Sem.07/Pract.07/References/Task02.cpp [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+const char UPPER_BORDER = 'Z';
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/Task02.cpp:2]: this constants are not needed

File: SI/Sem.07/Pract.07/References/Task02.cpp [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+const char LOWER_BORDER = 'z';
+const char UPPER_BORDER_TO_UP = 'A';
+const char LOWER_BORDER_TO_SMALL = 'a';
+bool toUpper(char& a)
+{
+	if (a >= LOWER_BORDER_TO_SMALL && a <= LOWER_BORDER)
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/Task02.cpp:8]: if-else is not needed.
Revise this solution.

File: SI/Sem.07/Pract.07/References/Task02.cpp [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+}
+bool toLower(char& b)
+{
+	if (b >= UPPER_BORDER_TO_UP && b <= UPPER_BORDER)
+	{
+		b += 32;
+		return true;
+		
+	}
+
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/Task02.cpp:27]: if we are not in the if , what do we return?

File: SI/Sem.07/Pract.07/References/Task03.cpp [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+	{
+		c = a;
+		a = tempC;
+	}
+	else if (b > c)
+	{
+		c = b;
+		b = tempC;
+	}
+
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/Task03.cpp:27]: remove unnecessary empty lines.

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
When you address comments put appropriate commit messages.

**Pull request closed without merging**